README.md

```diff
@@ -55,3 +55,9 @@ TODO non-functional:
 - document the release process, testing, debugging in the README
 - document how to run examples
 - acceptance tests
+
+
+### Documentation
+
+To generate documentation invoke:
+`go generate`
```

Comment:
I have following output:

```
➜  terraform-provider-kentik-synthetics git:(pstroz/documentation) ✗ go generate
no required module provides package github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs; to add it:
	go get github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs
main.go:1: running "go": exit status 1
```

To resolve that, add this development dependencies to tools/tools/go: 
`_ "github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs"`
Then run go mod tidy

See: https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module

Reply:
Done